tests/generation/Bears.py

```diff
@@ -130,6 +131,23 @@ def test_filter_relevant_bears_simple(self):
         self.assertTrue(len(res["C"]) > 0)
         self.assertTrue(len(res["Python"]) > 0)
 
+    def test_filter_relevant_bears_green_mode(self):
```

Comment:
The coverage reports show the following lines and transitions are still not reachable through the test cases:

80-82, 110, 79->80, 84->85, 109->110

Reply:
Nevermind, there are other lines (which are not part of this PR) in the `coala_quickstart.main` function that are unreachable too. So created a separate issue for that https://github.com/coala/coala-quickstart/issues/254